scripts/tests.sh

```diff
@@ -52,17 +52,31 @@ cleanup() {
 }
 trap cleanup INT TERM
 
-if [ "$1" = --junit_report ]
-then
-    if [ -z "$2" ]
-    then
-        echo "Usage: $0 [--junit_report <report_directory>]"
-        exit 1
-    fi
-    log_directory="$2"
-else
-    log_directory=""
-fi
+log_directory=""
+no_smt=""
+while [[ $# -gt 0 ]]
+do
+    case "$1" in
+        --junit_report)
+            if [ -z "$2" ]
+            then
+                echo "Usage: $0 [--junit_report <report_directory>] [--no-smt]"
+                exit 1
+            else
+                log_directory="$2"
+            fi
+            shift
+            shift
+            ;;
+        --no-smt)
+            no_smt="--no-smt"
```

Comment:
We already have a variable for SMT arguments.
```suggestion
            SMT_FLAGS+=(--no-smt)
```

Reply:
Ah, I see you're passing it down to `cmdlineTests.sh` too. Then I'd do this:
```suggestion
            no_smt="--no-smt"
            SMT_FLAGS+=(--no-smt)
```
I think it's better to have all SMT-related flags for soltest in `SMT_FLAGS`.

Reply:
done